sigma/plugins/utility/other/define.py

```diff
@@ -0,0 +1,78 @@
+import discord
+import aiohttp
+from bs4 import BeautifulSoup
+
+async def define(cmd, message, args):
+
+    definition = ""
+    defNum = 1
+    hasPrinted = False
+    defURL = r"https://www.merriam-webster.com/dictionary/"
+
+    if not args:
+        await message.channel.send(cmd.help())
+        return
+
+    try: 
+        defNum = int(args[-1])
+        defURL += "%20".join(args[0:-1])
+    except:
+        defURL += "%20".join(args)
+
+    async with aiohttp.ClientSession() as session:
+        async with session.get(defURL) as resp:
+            rawDef = await resp.text()
+    
+    try:
+        rawDef = rawDef[rawDef.index("<!--JC: actual definitions-->")-10:]
+    except:
+        embed = discord.Embed(title=":exclamation: The information you entered does not correlate to a Dictionary Entry.", color=0xFF0000)
+        await message.channel.send(None, embed=embed)
+        return
+
+    rawDefArr = rawDef.split("<!--JC: actual definitions-->")
+    soupObj = BeautifulSoup(rawDefArr[defNum], "html.parser")
+
+    try:    
+        if '<span class="intro-colon">' not in str(soupObj):
+            raise IndexError("The website has changed, please fix.")
+
+        for tag in soupObj.find_all(class_="intro-colon"):
+            definition += tag.text
+            if(len(tag.next_sibling.strip()) == 0):
+                if hasPrinted:
+                    continue
+                for tag2  in soupObj.find_all(class_="sx-link sc"):
+                    defCopy = definition
+                    definition += tag2.text
+                    if defCopy is definition:
+                        raise IndexError("The website has changed, please fix.")
```

Comment:
This logic has the same mistake as the previous one you fixed; If it finds the "sx-link sc" tag, then definition will not be same after adding the contents inside the tag, and so `if defCopy is definition` is always false.